app/views/sitemaps/index.xml.erb

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+   <url>
+      <loc><%= @host %></loc>
+      <lastmod><%= Time.now.strftime("%Y-%m-%dT%H:%M:%S")%></lastmod>
+   </url>
+  
+   <% @steps.each do |step| %>
+     <url>
+        <loc><%= "#{@host}/registrations/#{step.underscore}" %></loc>
```

Comment:
Can this use the `_url` helpers with host: ... ?, eg

```erb
<%= registrations_path step.underscore, host: @host %>
```

Then it will get updated automatically if the path is changed

Reply:
Good point

Reply:
Are you certain about the `skip_before_action`, I have seen other examples which reference it?

Reply:
Sorry ignore the above comment.